ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -259,7 +269,7 @@ const Annotations = React.memo(({ variant, mainGeneId, showMainGene }) => {
           <Transcripts variant={variant} />
         </Modal>
       )}
-      {svType && end && (
+      {svType && end && svType !== TRANSLOCATION_SV && (
```

Comment:
we really do not want to special case translocation where it isn't neccesssary. In this case, what we actually care about is whether `end !== pos`, as we never want to show a size of 0

Reply:
actually, you may want to change this to `svType && end && !endChrom` so it displays properly for those complex SVs we were discussing as well. In any case where there is an `endChrom` we probably can't accurately compute a size